graphql/ui_plugin_build_baron.go

```diff
@@ -192,6 +192,11 @@ func GetSearchReturnInfo(taskId string, exec string) (*thirdparty.SearchReturnIn
 	buildBaronProjects := BbGetConfig(settings)
 	bbProj, ok := buildBaronProjects[t.Project]
 
+	// if the project uses custom ticket creation, don't use the default search
+	if bbProj.TaskAnnotationSettings.FileTicketWebHook.Endpoint != "" {
+		return nil, bbConfig, errors.Errorf("Build Baron project for %s is configured to use a custom webhook", t.Project)
```

Comment:
Should this error or just not return anything? Also, this should be after the !ok

Reply:
I was conflicted about that, but I checked to make sure that it doesn't break our current implementation in Spruce, so I think for now an error makes sense. If someone hits the error, it'll probably mean they changed something and prompt them to investigate.
You are right about it being after !ok. I'll move it.

Reply:
This still needs to be moved.

Reply:
Apologies, I forgot to push my last commit. Should be good now.